specification/billing/resource-manager/Microsoft.Billing/stable/2023-04-01/billingAccount.json

```diff
@@ -1,66 +1,115 @@
 {
   "swagger": "2.0",
   "info": {
-    "description": "Billing client provides access to billing resources for Azure subscriptions.",
     "title": "BillingManagementClient",
+    "description": "Documentation for Microsoft.Billing.",
+    "contact": {
+      "name": "Microsoft.Billing",
+      "url": "https://learn.microsoft.com/en-us/rest/api/billing/",
+      "email": "cabbpt@microsoft.com"
+    },
     "version": "2023-04-01"
   },
-  "host": "management.azure.com",
-  "schemes": [
-    "https"
-  ],
-  "consumes": [
-    "application/json"
-  ],
-  "produces": [
-    "application/json"
-  ],
   "paths": {
     "/providers/Microsoft.Billing/billingAccounts": {
       "get": {
+        "tags": [
+          "BillingAccount"
+        ],
         "description": "Lists the billing accounts that a user has access to.",
         "externalDocs": {
           "url": "https://docs.microsoft.com/en-us/rest/api/billing/"
         },
         "operationId": "BillingAccounts_List",
+        "produces": [
+          "application/json"
+        ],
         "parameters": [
           {
-            "$ref": "./types.json#/parameters/apiVersionParameter"
+            "in": "query",
+            "name": "includeAllBillingAccounts",
+            "description": "When true, results will include Billing Accounts that the user does not have a direct role assignment on if the user has one of the following AAD roles: Global Administrator, Global Reader, Billing Administrator.",
+            "type": "boolean",
+            "default": false
           },
           {
-            "description": "May be used to expand the soldTo, invoice sections and billing profiles.",
             "in": "query",
-            "name": "$expand",
-            "required": false,
+            "name": "excludeAccountsWithoutBillingProfiles",
+            "description": "When true, results will exclude Billing Accounts that are not fully created if the user has one of the following AAD roles: Global Administrator, Global Reader, Billing Administrator.",
+            "type": "boolean",
+            "default": true
+          },
+          {
+            "in": "query",
+            "name": "includeDeletedBillingAccounts",
```

Comment:
Here we're calling it "includeDeletedBillingAccounts"
In Billing Profiles, we're calling it "includeDeleted"

Let's pick one convention